src/modules/shared/types/visit.ts

```diff
@@ -18,14 +18,16 @@ export interface Visit {
 	spaceImage?: string;
 	spaceLogo?: string;
 	spaceInfo?: string;
+	spaceDescription?: string;
+	spaceServiceDescription?: string;
 	startAt?: string;
 	status: VisitStatus;
-	timeframe: string;
+	timeframe?: string;
 	updatedAt: string;
 	userProfileId: string;
 	visitorId: string;
 	visitorMail: string;
-	visitorName: string;
+	visitorName?: string;
```

Comment:
Why is visitor name optional?
isn't this always filled in?
If not, that might be a bug in the backend

Reply:
This typing allows the `VisitSummary` to inherit properties from the `Visit` type while also ensuring it can conditionally not render certain fields if they're forced to be empty